result.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<head>
+    <title>yummy</title>
+    <meta charset="utf-8">
+    <script type="text/javascript"
+        src="//dapi.kakao.com/v2/maps/sdk.js?appkey=f6f0b7d7340ad72b589bf8056b7240a6&libraries=services"></script>
+    <link rel="stylesheet" type="text/css" href="index_style.css">
+</head>
+
+<body>
+    <div id="title">
+        맛.zip
+        <div id="topMenu">
+            <a href="/index.html">home</a>
+            <a href="/serch.html">list</a>
+        </div>
+    </div>
+</body>>
```

Comment:
```suggestion
</body>
```
닫는 꺾은괄호가 한번 더 들어갔습니다.

Reply:
사용하지 않는 파일이라서 삭제 하였습니다!